platform-tests/src/test/java/org/junit/platform/console/tasks/DiscoveryRequestCreatorTests.java

```diff
@@ -282,6 +298,10 @@ private LauncherDiscoveryRequest convert() {
 		return creator.toDiscoveryRequest(options);
 	}
 
+	private void assertFilters(Filter<?> filter, String filteredElement) {
```

Comment:
Not sure if this is a good idea... Might need a better name 🤔

Reply:
`assertFilterDescriptionContains`?

Should we rather call the filter and check it includes/excludes the expected names?

Reply:
Indeed!